packages/protocol/scripts/bash/verify-deployed.sh

```diff
@@ -32,4 +32,31 @@ done
 source scripts/bash/release-lib.sh
 build_tag $BRANCH $LOG_FILE
 
+if [ "$BRANCH" = "core-contracts.v10" ]; then
+  if [ ! -d "build/mento" ]; then
+    mkdir -p build/mento
+    cd build/mento
+    git clone --depth 1 --branch v2.2.1 https://github.com/mento-protocol/mento-core.git
+    cd mento-core
+    yarn 
+    forge install
+    forge build
+  else 
+    cd build/mento/mento-core
+  fi
+
+  # Replace the bytecode of the SortedOracles contracts with the bytecode from Mento core v2.2.1
+  orig_value_SortedOracles=$(jq -r '.deployedBytecode.object' out/SortedOracles.sol/SortedOracles.json)
+  substring_to_replace='__$e9f4a9f9de32ce6d7070252f1b707ecbd2$__'
```

Comment:
If we keep this, would be good to leave some explainer comments for where this came from.

Reply:
Comment added